h2o-r/tests/testdir_algos/glm/runit_GLM_makeGLMModel.R

```diff
@@ -16,6 +16,7 @@ test <- function(h) {
     model <- h2o.glm(x = x, y="CAPSULE", training_frame=prostate_h2o,family="binomial", alpha=.99,lambda=.2)
     new_beta <- c(-7.27968, -0.01213, -0.62424, 0.55661, 0.48375, 0.02739, -0.01124, 0.97632)
     names(new_beta) <- c("Intercept","AGE","RACE","DPROS","DCAPS","PSA","VOL","GLEASON")
+    browser()
```

Comment:
@wendycwong please remove

Reply:
Let me remove it.

Reply:
Done in https://github.com/h2oai/h2o-3/pull/4301/commits/9ec799065fc08d6f8bb0d5838bd355f7c3cfb2fa. :heavy_check_mark: